src/Uno.UI.sln

```diff
@@ -346,7 +346,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.UI.Svg.Skia", "AddIns\U
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.UI.Svg.Reference", "AddIns\Uno.UI.Svg\Uno.UI.Svg.Reference.csproj", "{4C572A79-0878-4841-82E1-B3125B03036B}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Uno.UI.RemoteControl.Messaging", "Uno.UI.RemoteControl.Messaging\Uno.UI.RemoteControl.Messaging.csproj", "{8E3C8617-ABC2-464C-9376-1B334250F80F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.UI.RemoteControl.Messaging", "Uno.UI.RemoteControl.Messaging\Uno.UI.RemoteControl.Messaging.csproj", "{8E3C8617-ABC2-464C-9376-1B334250F80F}"
```

Comment:
I'm not sure of the changes in the solution file.

Reply:
Undone